tox.ini

```diff
@@ -121,6 +121,19 @@ extras =
 commands =
     pytest {tty:--color=yes}  {posargs} {[testenv]covargs} test/integration
 
+[testenv:minmax_rows]
```

Comment:
Super useful! Would be helpful to note this change in the description of the PR. 
Do you want to reference the addition of this test in the PUDL documentation, "Testing the ETL"?

Reply:
I added this as a convenience (and we should have a long time ago) but I think it may be rendered obsolete almost immediately by the dagster merge, so I didn't want to spread it around just yet. We should figure out how to do something similar after the merge happens.

But I could be totally wrong here. @zschira would this work under the dagster setup?